crates/cheatcodes/src/inspector.rs

```diff
@@ -893,107 +907,116 @@ impl Cheatcodes {
         }
 
         if self.use_zk_vm {
-            info!("running create in zk vm");
-            if input.init_code().0 == DEFAULT_CREATE2_DEPLOYER_CODE {
-                info!("ignoring DEFAULT_CREATE2_DEPLOYER_CODE for zk");
-                return None
-            }
+            if self.skip_zk_vm {
```

Comment:
perhaps we can avoid this right indenting by returning early, just like we are doing for the `DEFAULT_CREATE2_DEPLOYER_CODE` (and also what we do when `self.use_zk_vm` is false in call)

Reply:
Had that earlier but that either makes duplicate checks `self.use_zk && self.skip_zk_vm` or an early return which can cause issues if code is added below (unsure if it's okay to return `None` from within the block and not cause issues if some code is added later.